crates/typed-store/src/rocks/mod.rs

```diff
@@ -440,25 +431,38 @@ pub fn open_cf_opts<P: AsRef<Path>>(
     db_options: Option<rocksdb::Options>,
     opt_cfs: &[(&str, &rocksdb::Options)],
 ) -> Result<Arc<rocksdb::DBWithThreadMode<MultiThreaded>>, TypedStoreError> {
+    println!("BEGIN");
     // Customize database options
     let mut options = db_options.unwrap_or_default();
 
-    let mut opt_cfs: std::collections::HashMap<_, _> = opt_cfs.iter().cloned().collect();
-    let cfs = rocksdb::DBWithThreadMode::<MultiThreaded>::list_cf(&options, &path)
+    let opt_cfs: std::collections::HashMap<_, _> = opt_cfs.iter().cloned().collect();
+    /*
```

Comment:
Did you mean to leave this commented? If so use single line comments. Multi line comments make it difficult to see if code is live or dead when using grep.

Reply:
wait what the heck this wasn't in the change i pushed....